review/migrations/0001_initial.py

```diff
@@ -1,16 +1,17 @@
-# Generated by Django 4.0.3 on 2022-03-29 21:22
+# Generated by Django 4.0.3 on 2022-04-11 07:33
```

Comment:
You're right, it was done because I re-created the migrations for the review app, next time I will just run the migrate command in order to update the changes in the data migrations.

Reply:
@jeniaSakirko is right. You need to revert the change to this migration and run the migrate command in order for this PR to be able to merge.

Reply:
Done